SparkPost/matchbox
# UX-581 Fix Inline child being excluded from Banner component

### What Changed

- Updated `excludeChild` function to not check `name`, only `displayName`

### How To Test or Verify

- Testing this one is a little challenging, as it was only happening in 2web2ui
- Make sure you're using this version of Matchbox in 2web2ui
- `npm run build` in 2web2ui
- `npm run serve` and check that the `Inline` component isn't being excluded

### PR Checklist
